README.md

```diff
@@ -1,3 +1,7 @@
+<div align="center" style="padding: 30px 0px 20px 0px;">
+  <img src="https://github.com/user-attachments/assets/95171e10-a4cd-474e-9bb2-c95524f39b6a" alt="Logo" width=250>
```

Comment:
```suggestion
  <img src="https://github.com/user-attachments/assets/4cb9fb46-6c96-4ac0-b7f9-8560e44e11d1" alt="Logo" width=250>
```

We worked together offline on this updated image. This is the image shared [in this comment](https://github.com/thoughtbot/belt/pull/50#pullrequestreview-2254560561).